CascadeEnergy/monolog-kinesis-handler
# Version 0.1 -- Initial implementation

To test:
- Review unit tests and ensure coverage
- Verify that the example in the README.md file is correct, and that log messages reach a Kinesis stream when using the example code
